src/lib/logstore/log_dev.cpp

```diff
@@ -88,7 +88,7 @@ void LogDev::start(bool format, JournalVirtualDev* vdev) {
             m_store_found_cb(spair.first, spair.second);
         }
 
-        THIS_LOGDEV_LOG(INFO, "get start vdev offset during recovery {} log indx {} ",
+        THIS_LOGDEV_LOG(INFO, "family {} get start vdev offset during recovery {} log indx {} ", m_family_id,
```

Comment:
You can change THIS_LOGDEV_LOG to add the family in everyone so we don't need to repeat it everywhere?

Reply:
I checked it, it was already there, just added because couldnt see when testing.